pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -487,6 +505,11 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 	lbBackendPoolName := getBackendPoolName(clusterName)
 	lbBackendPoolID := az.getBackendPoolID(lbName, lbBackendPoolName)
 
+	lbIdleTimeout, err := getIdleTimeout(service)
```

Comment:
so what if there is no `service.beta.kubernetes.io/azure-load-balancer-idle-timeout` setting, it will `return nil, nil`, are we going to handle that? And what if it's a negative?

Reply:
For validation of the value beyond if it converts to an int... I didn't want to do too much validation here as I expect the backend to deal with this, but I do see value in validating here since they are fairly hard values that are well documented (or at least were easy for me to find), which saves us a failed request.

Reply:
This is all updated

Reply:
Thanks. The range is [4,30]